### PR TITLE
linux-filetracer: add support for linux kernel 5.12+

### PR DESCRIPTION
**Pull request by disaykin**

### CODE REVIEW

File: src/libdrakvuf/linux.c [hunk 1]
```diff
@@ -245,6 +245,69 @@
...
+        if (!version)
+        {
+            PRINT_DEBUG("Failed to extract linux kernel version\n");
+            return NULL;
+        }
+
+        int major = 0, minor = 0, patch = 0;
+        sscanf(version, "%d.%d.%d", &major, &minor, &patch);
+
+        drakvuf->kernel_ver.major = major;
```
[the remark this answers is not in the document]
Comment (tklengyel) [reply src/libdrakvuf/linux.c:301]: Replace `sscanf` with `g_strdup_printf`

[the remark this answers is not in the document]
Comment (tklengyel) [reply src/libdrakvuf/linux.c:301]: CodeQL is complaining about sscanf return not being checked

**Comment by disaykin** [comment]
@tklengyel Cppcheck reports too many unrelated warnings.

**Comment by tklengyel** [comment]
You can ignore cppcheck, ubuntu-latest jumped to 22.04 and cppcheck version is increased and it is spotting things it hadn't before.

**Comment by tklengyel** [comment]
@drakvuf-jenkins Retest this please

**Pull request merged**